### ISSUE TITLE
[Cognitive Services] Error unmarshaling face API response

### ISSUE THREAD

**Issue by dexyk** [Azure/azure-sdk-for-go#886]
When I try Face API I get error unmarshalling `float` into `int`. I think the reason is in the file [models.go](https://github.com/Azure/azure-sdk-for-go/blob/2592daf71ab6b95dcfc7f7437ecc1afb9ddb7360/dataplane/cognitiveservices/face/models.go)
```type Landmarks struct {
	PupilLeft           *Position `json:"pupilLeft,omitempty"`
	PupilRight          *Position `json:"pupilRight,omitempty"`
	NoseTip             *Position `json:"noseTip,omitempty"`
	MouthLeft           *Position `json:"mouthLeft,omitempty"
```
Position contains `ints` and `faceLandmarks` in Face API response contain `floats`:
```
"faceLandmarks": {
      "pupilLeft": {
        "x": 504.8,
        "y": 206.8
      },
      "pupilRight": {
        "x": 602.5,
        "y": 178.4
      },
```

**Comment by vladbarosan** [comment]
Hi @dexyk , thanks for opening the issue. We will look to fix this for the next release.

**Comment by vladbarosan** [comment]
@dexyk  we just released v0.12  this is fixed there. Also please note, `dataplane` will be deprecated. Latest changes are in `services` folder. Please see here : https://github.com/Azure/azure-sdk-for-go/blob/dev/services/cognitiveservices/v1.0/face/models.go
